packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PrimaryButton, Stack, Text } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+import React, { useRef, useState } from 'react';
+import { CompositeLocale, useLocale } from '../../localization';
+/* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+import { HoldButton } from '@internal/react-components';
+/* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+import { usePropsFor } from '../hooks/usePropsFor';
+import {
+  holdPaneContentStyles,
+  holdPaneLabelStyles,
+  holdPaneTimerStyles,
+  paneStyles,
+  resumeButtonStyles
+} from '../styles/HoldPane.styles';
+
+/**
+ * Hold pane to display when the user places themselves on hold
+ *
+ * @beta
+ */
+export const HoldPane = (): JSX.Element => {
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const holdButtonProps = usePropsFor(HoldButton);
+  const locale = useLocale();
+
+  const strings = stringsTrampoline(locale);
+
+  const [time, setTime] = useState<number>(0);
+
+  const elapsedTime = getReadableTime(time);
+
+  const startTime = useRef(performance.now());
+
+  React.useEffect(() => {
+    const interval = setInterval(() => {
+      setTime(performance.now() - startTime.current);
+    }, 10);
+    return () => {
+      clearInterval(interval);
+    };
+  });
```

Comment:
checking if expected: this has no dependency array so will be executed every render pass